packages/settingeditor/style/base.css

```diff
@@ -320,18 +277,34 @@
   height: 100%;
 }
 
+.jp-SettingsForm .jp-Buttonbar {
```

Comment:
Let's discuss whether we want to have button bar in https://github.com/jupyterlab/jupyterlab/issues/14403. on CSS-level, I would suggest to replace `jp-Buttonbar` with a more specific `jp-SettingsForm-buttonbar` to avoid introducing a new unscoped class which is only used in a single place (and for performance reasons) - but this can be done in a separate PR